src/handlers/transfers/FxFulfilService.js

```diff
@@ -218,23 +288,39 @@ class FxFulfilService {
   }
 
   async validateFulfilment(transfer, payload) {
-    const isValid = this.validateFulfilCondition(payload.fulfilment, transfer.ilpCondition)
+    const isValid = this.validateFulfilCondition(
+      payload.fulfilment,
+      transfer.ilpCondition
+    )
 
     if (!isValid) {
       const fspiopError = fspiopErrorFactory.fxInvalidFulfilment()
-      const apiFSPIOPError = fspiopError.toApiErrorObject(this.Config.ERROR_HANDLING)
+      const apiFSPIOPError = fspiopError.toApiErrorObject(
+        this.Config.ERROR_HANDLING
+      )
       const eventDetail = {
         functionality: Type.POSITION,
         action: Action.FX_ABORT_VALIDATION
       }
-      this.log.warn('callbackErrorInvalidFulfilment', { eventDetail, apiFSPIOPError, transfer, payload })
-      await this.FxTransferModel.fxTransfer.saveFxFulfilResponse(transfer.commitRequestId, payload, eventDetail.action, apiFSPIOPError)
+      this.log.warn('callbackErrorInvalidFulfilment', {
+        eventDetail,
+        apiFSPIOPError,
+        transfer,
+        payload
+      })
+      await this.FxTransferModel.fxTransfer.saveFxFulfilResponse(
+        transfer.commitRequestId,
+        payload,
+        eventDetail.action,
+        apiFSPIOPError
+      )
 
       await this.kafkaProceed({
         consumerCommit,
         fspiopError: apiFSPIOPError,
         eventDetail,
-        messageKey: transfer.counterPartyFspTargetParticipantCurrencyId.toString()
+        messageKey:
```

Comment:
do you think, after such reformatting the code looks better?

Reply:
@geka-evk Could be the linter in the pre-commit hook. Will fix that.